lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -59,55 +62,11 @@ class KubernetesMetadataFilter < Fluent::Filter
 
     config_param :annotation_match, :array, default: []
     config_param :stats_interval, :integer, default: 10
+    config_param :namespace_watch_strategy, :string, default: 'default'
+    config_param :pod_watch_strategy, :string, default: 'default'
```

Comment:
there is an `enum` type - see https://github.com/ViaQ/fluent-plugin-viaq_data_model/blob/master/lib/fluent/plugin/filter_viaq_data_model.rb#L113 e.g.
```ruby
   config_param :namespace_watch_strategy, :enum, list: [:default], default: :default
```
the value is a sym, not a string - use it like this
```ruby
  if @namespace_watch_strategy == :default
    ...
```